cypress/e2e/sample_test_spec/SampleTestRequest.cy.ts

```diff
@@ -0,0 +1,88 @@
+import LoginPage from "pageobject/Login/LoginPage";
+import { PatientConsultationPage } from "pageobject/Patient/PatientConsultation";
+import { PatientPage } from "pageobject/Patient/PatientCreation";
+import { SampleTestPage } from "pageobject/Sample/SampleTestCreate";
+
+describe("Sample Test", () => {
+  const sampleTestPage = new SampleTestPage();
+  const patientPage = new PatientPage();
+  const loginPage = new LoginPage();
+  const patientConsultationPage = new PatientConsultationPage();
+  const patientName = "Dummy Patient 11";
+  const sampleTestType = "BA/ETA";
+  const icmrCategory = "Cat 0";
+  const icmrLabel = "Test Icmr Label";
+  const doctorName = "Dr John Doe";
+  const atypicalDetails = "Patient showing unusual symptoms";
+  const diagnosis = "Suspected respiratory infection";
+  const etiologyIdentified = "Bacterial infection suspected";
+  const differentialDiagnosis = "Possibly a viral infection";
+  const fastTrackReason =
+    "The patient has a high risk of complications and requires immediate testing.";
+  const sampleTestStatus = "Request Submitted";
+  const expectedSampleTestType = "ba/eta";
+  const sampleTestResult = "Awaiting";
+
+  before(() => {
+    loginPage.loginAsDistrictAdmin();
+    cy.saveLocalStorage();
+  });
+
+  beforeEach(() => {
+    cy.restoreLocalStorage();
+    cy.clearLocalStorage(/filters--.+/);
+  });
+
+  it("should request a new sample test", () => {
+    // Ensure patient list API is loaded before proceeding
+    cy.awaitUrl("/patients");
+    patientPage.visitPatient(patientName);
+    sampleTestPage.interceptPatientDetailsAPI();
+    patientConsultationPage.clickPatientDetails();
+    sampleTestPage.verifyPatientDetailsResponse();
```

Comment:
this API you're trying verify for the function (Clicking Patient Details Button) from patientconsultationpage right, so the related API verifying function should be also kept in that pageobject files, so move it from sampletestpage object to patientconsultationpage object file